nethserver-tancredi.spec

```diff
@@ -48,6 +48,7 @@ perl createlinks
 install NethVoiceAuth.php %{buildroot}/usr/share/tancredi/src/Entity/
 install AsteriskRuntimeFilter.php %{buildroot}/usr/share/tancredi/src/Entity/
 install migration.php  %{buildroot}/usr/share/tancredi/scripts/
+mkdir -p %{buildroot}/usr/sbin
```

Comment:
If you create it before `%{genfilelist}` the resulting RPM will own `/usr/sbin`: very bad!

```suggestion
```